config/config.go

```diff
@@ -169,12 +168,8 @@ type FeeUnitConfig struct {
 
 // MinerAPI holds connection info for a single miner endpoint
 type MinerAPI struct {
```

Comment:
Maybe let's name it: ArcAPI instead -> Miner was a name from mAPI (Miner API).

Reply:
MinerAPI changed to ArcAPI